### PR TITLE
transpose: add .md and .yml

### PR DESCRIPTION
**Pull request by andrey-gvrd**
In reference to rust implementation <https://github.com/exercism/xrust/pull/151>

### CODE REVIEW

**Comment by petertseng** [comment]
The examples given seem to imply that the elements of the input should be transposed word-wise. For example, that:

```
Aaa Bbb Ccc
Ddd Eee Fff
``` 

would tranpose to;

```
Aaa Ddd
Bbb Eee
Ccc Fff
```

If I understand the test expectations in https://github.com/exercism/xrust/pull/151, it appears the transposition is instead character-wise. It would be good to make this clear in the readme so that there is no confusion. especially because character-wise may indicate that

```
A B C
D E F
```

is supposed to actually transpose to:

```
AD

BE

CF
```

which is not the same as what the example indicates.

---

Finally, it would indeed be good to get a file with common test cases.

**Comment by andrey-gvrd** [comment]
Fixed the example to remove the ambiguity.

File: transpose.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+```
+ABC
+DEF
+```
+
+is given by:
+
+```
+A D
```
Comment (petertseng) [inline transpose.md]: why space?

**Comment by IanWhitney** [comment]
I think this is ready. Will merge on Monday, July 18 unless someone says otherwise.

**Comment by kytrinyx** [comment]
Thank you!

**Pull request merged**